### DISCUSSION TITLE
Does the nextjs repo also contain the code for the nextjs.org website

### DISCUSSION THREAD

**Discussion by SamarthBagga** [vercel/next.js#49301] [Help]
I am new to open source and would love to the start contributing to nextjs as it is something that I have used a lot. 
I want to contribute to the nexr website nextjs.org. Could someone guide to me to the right repo for the site or is the vercel/next repo, the correct one ?

**Comment by icyJoseph** [comment]
If I remember correctly, that site is closed source. Same for the docs site, though some of the docs are here: https://github.com/vercel/next.js/tree/canary/docs

For the learning material you can open issues here: https://github.com/vercel/next-learn, but it is not the source code for the learning site.

**Reply by SamarthBagga** [reply]
I found an open issue on the nextjs repo pertaining to a faulty link on the nextjs site, so is there no way for me to contribute by correcting that link?

**Reply by icyJoseph** [reply]
Could you share what it was? And I guess not, there used to be a feedback option on the site but it is gone.

**Reply by SamarthBagga** [reply]
yes sure ! So it was an issue with a link on nextjs.org in the documentation section redirecting to a wrong page instead of the page it should.
